src/schematics/ng-add/_files/src/main.single-spa.ts.template

```diff
@@ -13,8 +13,7 @@ if (environment.production) {
 }
 
 export default singleSpaAngular({
-  mainModule: AppModule,
-  angularPlatform: platformBrowserDynamic(),
+  bootstrapFunction: () => platformBrowserDynamic().bootstrapModule(AppModule),
```

Comment:
This is what makes the bundle size go down -- Angular can statically analyze this code and replace platformBrowserDynamic with platformBrowser, which is smaller and intended for production builds.

Reply:
I'm guessing that this will also remove the warning when `ng build`ing a single spa app? Looks good to me then :-)

Reply:
Yep! That warning was there for the same reason

Reply:
This brought the build size of one of my larger apps down 500KB and got rid of the `Lazy routes discovery is not enabled. Because there is neither an entryModule nor a statically analyzable bootstrap code in the main file.` warning, good stuff!